supervisor_types_test.go

```diff
@@ -183,3 +183,88 @@ func TestIngestionSpecWithTypedDimensions_MarshalJSON(t *testing.T) {
 		require.Equal(t, string(expected), string(actual), fmt.Sprintf("expected: %s\nactual: %s", string(expected), string(actual)))
 	})
 }
+
+var jsonWithSqlInputSource = `{
```

Comment:
qq: was this ingestion spec tested on druid? I'm wondering how for shall we go with automation here.

Reply:
Not yet. I plan to write e2e tests with records going into Postgres and then migrated into the druid.